docs: clarify caching CMM initialization parameters (#176)
awslabs/aws-encryption-sdk-specification: framework/caching-cmm.md

## Initialization

On caching CMM initialization,
the caller MUST provide the following values:

- [Underlying Cryptographic Materials Cache (CMC)](#underlying-cryptographic-materials-cache)
- [Underlying Cryptographic Materials Manager (CMM)](#underlying-cryptographic-materials-manager)
- [Cache Limit TTL](#cache-limit-ttl)

Additionally, the caching CMM MUST define the following:

> 

Reviewer: Per our latest wording, these should be "the caching CMM MUST optionally accept"

## Initialization

On caching CMM initialization,
the caller MUST provide the following values:

- [Underlying Cryptographic Materials Cache (CMC)](#underlying-cryptographic-materials-cache)
- [Cache Limit TTL](#cache-limit-ttl)

Additionally, the caller MUST provide one of the following values:

- [Underlying Cryptographic Materials Manager (CMM)](#underlying-cryptographic-materials-manager)
- [Keyring](keyring-interface.md)

If the caller provides a keyring,
then the caching CMM MUST set its underlying CMM

> to a [default CMM](../../framework/default-cmm.md) that wraps the keyring.

Reviewer 2: ```suggestion
to a [default CMM](../../framework/default-cmm.md) initialized with the keyring.
```
